css/properties/place-self.json

```diff
@@ -1,12 +1,57 @@
-{
+  {
```

Comment:
What's this change and why isn't it failing the lint? 🧐

Reply:
Weird. Fixed in next commit.